superset/views/core.py

```diff
@@ -810,6 +810,14 @@ def shortner(self):
 
 appbuilder.add_view_no_menu(R)
 
+class Reactcode(BaseSupersetView):
+    @expose('/')
+    def index(self):
+        # return redirect("/reactcode/index.html")
+        return self.render_template("custom/reactcode/index.html")
+        # return self.render_template("custom/reactcode/index.html")
+
+appbuilder.add_view_no_menu(Reactcode)
 
 class Superset(BaseSupersetView):
```

Comment:
Here we are implementing views in a fixed manner but in actual scenario there may  be be more than one app deployed with superset ,so do we need to write view  and corresponding API expose  for each app

Reply:
Ideally no. We should be able to handle that through url-id passed to index method.
That was not done in this poc.

In next stage when we integrate another angular app all this piece s should come together